src/components/chatPage/ChatHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { BackArrowIcon, FindIcon, CallIcon, IndexIcon } from '../../assets';
+import { flexCenter } from '../../styles/GlobalStyle';
+import userData from '../../assets/data/userData.json';
+
+interface ChatHeaderProps {
+  userId: number;
+  setUserId: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export default function ChatHeader(props: ChatHeaderProps) {
+  const { userId, setUserId } = props;
+
+  function handleUser() {
+    const newUserId = userId === 0 ? 1 : 0;
+    setUserId(newUserId);
+  }
+
+  return (
+    <Wrapper>
+      <Left>
+        <BackArrowIcon />
+        <SentName onClick={handleUser}>{userData.data.find((user) => user.id === userId)?.name}</SentName>
+      </Left>
+      <Right>
+        <FindIcon />
+        <CallIcon />
+        <IndexIcon />
+      </Right>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.section`
+  display: flex;
+  justify-content: space-between;
+
+  width: 100%;
+  padding: 0.8rem;
+
+  background-color: ${({ theme }) => theme.colors.blue_bg};
+`;
+
+const Left = styled.span`
+  ${flexCenter}
+  gap: 0.4rem;
+
```

Comment:
back icon이랑 사용자랑 같은 높이에 두기 위해 아래 부분 추가해주시면 좋을 것 같아요~
```suggestion
display : flex;
```